src/Assets/TestProjects/TestConsoleApp/ConsoleApp.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
For simple projects like this, I prefer the `TestProject` APIs rather than adding new assets under the TestProjects folder.  You can set project properties and source files in code.

Can we switch to doing that instead of adding more TestProjects assets?

Reply:
These are a bit different in that the *.cs files don't have any using statements. Though it's possible to modify *.cs files in code it's not well supported by `TestAsset` I think.

Reply:
You can use the `SourceFiles` property of `TestProject`:

```c#
            testProject.SourceFiles["Program.cs"] = @"
//  Top level statement
Console.WriteLine(""Hello World!"");
```

Reply:
I'll update the tests after I figure out the language version issues.